src/Cake.Common.Tests/Unit/Tools/NuGet/Pack/NuGetPackerTests.cs

```diff
@@ -328,7 +328,14 @@ public void Should_Replace_Template_Tokens_In_Nuspec()
                     RequireLicenseAcceptance = true,
                     Copyright = "The copyright",
                     ReleaseNotes = new[] { "Line #1", "Line #2", "Line #3" },
-                    Tags = new[] { "Tag1", "Tag2", "Tag3" }
```

Comment:
Would prefer two extra tests that validates the new `Files` functionality. One with and one without XML namespace (this was a problem that was encountered).

Maybe also test with some other files to make sure that the nuspec transformation isn't being done by the old transformator, where the values were hard coded into the nuspec (functionality we want to preserve).